src/locales/en.json

```diff
@@ -209,6 +209,9 @@
       "testnet": "Testnet"
     },
     "statistic": {
+      "transaction_num": "Transaction Num",
```

Comment:
```suggestion
      "transaction_num": "Transaction Count",
```

Reply:
I just copied this from the design draft, may @Sven-TBD and @Kirl70 could have a look